### PR TITLE
Alien corpse fix #12311

### LINKED ISSUE #12311: Can't debrain aliens
Attempting to use a scalpel or circular saw on them after applying the drapes to their head and selecting 'organ manipulation' just results in attempting to butcher them

### PR DESCRIPTION
**Pull request by bear1ake**
Also new alien corpses and new simple alien types.

### CODE REVIEW

**Comment by WJohn** [comment]
This is a pretty major balancing issue with the corpses. Some away missions have fake xenos like MO19 (and I suspect that it'd be likely that we run into these more often in the future). This means that if you go around easily killing those and bringing them back to the station, you have easy access to level 8 biotech and all the xeno organs. This means stuff like super high-end implants as well as xeno immunity and abilities for relatively little effort.

I say we just close the actual issue since having them drop fake corpses might as well be intended for balance purposes.

**Comment by bear1ake** [comment]
@WJohn okay.

**Comment by KorPhaeron** [comment]
I don't think it's a big deal honestly. Xeno organ surgery takes a large amount of work, and its never ever done because in rounds with real xenos the shuttle is called (or everyone dies).

Letting simple xenos drop real bodies might let the surgery actually get used.

**Comment by WJohn** [comment]
Kor it really doesn't take work, though. The tough part of getting ahold of xeno brains is normally actually killing the xenos and still having a functioning station to do that in. Xenos are normally immensely dangerous and tend to control large parts of the station, so successfully killing them and profiting is the hard part. Having them be braindead-stupid simple animals removes the entire challenge, and gives you effectively free access to revivor implants and similarly overpowered items. 

It'd be like if there were away missions or pieces of space content with frequently spawning anomalies, you'd see phazons WAY more.

**Comment by Supermichael777** [comment]
Does this actually fix #12311 which was specifically about CARBON Xenos and not simple-mob ones?
And giving fake Xenos real organs is probably a feature not a fix.

File: code/modules/mob/living/simple_animal/corpse.dm [hunk 1]
```diff
@@ -10,6 +10,7 @@
 /obj/effect/landmark/mobcorpse
 	name = "Unknown"
 	var/mobname = "Unknown"  //Unused now but it'd fuck up maps to remove it now
+	var/mobtype = /mob/living/carbon/human	//Mob's type. Now uses for spawn dead mob from aliens
```
Comment (Supermichael777) [inline code/modules/mob/living/simple_animal/corpse.dm]: used for. rather irreverent but still

**Comment by Supermichael777** [comment]
yeah this doesn't touch code/modules/mob/living/carbon/alien/humanoid/humanoid where the problem is. Also butchering checks if it has butchering results directly without passing through any vars which is terrible and should be changed.

**Comment by KazeEspada** [comment]
The moon mission sucks. At least now it might have good rewards.

**Comment by Dorsisdwarf** [comment]
Saying your feature fixes a bug is a bad idea

**Comment by Shadowlight213** [comment]
:-1: on this. This removes pretty much all the challenge of getting the organs, and with the dead xeno corpse landmarks being real xenos, you don't even need to kill a stupid simple animal to get your op implants!

**Pull request closed without merging**